data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -1 +1,552 @@
+        icon_completed="units/human-outlaws/rogue.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S6_Liberator_of_Elensefar"
+        name= _ "Liberator of Elensefar"
+        name_completed= _ "Liberator of Elensefar"
+        description= _ "Liberate Elensefar from the clutches of the evil Queen!"
+        description_completed= _ "Liberated Elensefar from the clutches of the evil Queen!"
+        icon="data/campaigns/Heir_To_The_Throne/images/units/konrad-commander.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="data/campaigns/Heir_To_The_Throne/images/units/konrad-commander.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S7_Clash_at_the_Crossroads"
+        name= _ "Clash at the Crossroads"
+        name_completed= _ "Clash at the Crossroads"
+        description= _ "Defeat both orcish commanders in <i>Crossroads</i>"
+        description_completed= _ "Defeated both orcish commanders in <i>Crossroads</i>"
+        icon="units/orcs/warlord-defend-2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/orcs/warlord-defend-2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S8_Met_Your_Mirror"
+        name= _ "Met Your Mirror"
+        name_completed= _ "Met Your Mirror"
+        description= _ "Confront the scion of the Dark Queen"
+        description_completed= _ "Bested the scion of the Dark Queen"
+        icon="data/campaigns/Heir_To_The_Throne/images/units/human-princess.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="data/campaigns/Heir_To_The_Throne/images/units/human-princess.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S9_Unholy_Bane"
+        name= _ "Unholy Bane"
+        name_completed= _ "Unholy Bane"
+        description= _ "Purge the Undead while on the road towards Gryphon Mountain"
+        description_completed= _ "Purged the Undead while on the road towards Gryphon Mountain"
+        icon="units/unknown-unit.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/undead-necromancers/lich.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S10_Egg_Snatcher"
+        name= _ "Egg Snatcher"
+        name_completed= _ "Egg Snatcher"
+        description= _ "Steal the Mother Gryphon's Eggs"
+        description_completed= _ "Shamelessly Stole the Mother Gryphon's Egg"
```

Comment:
One line says Eggs, the other says its singular.

Also, both of them are using a non-typographical apostrophe.